### PR TITLE
Adapt for Runtastic

### PR DESCRIPTION
**Pull request by rtgeha**
Initial adaptations for Runtastic based on our Brainstorming session

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
     grid: "#bbb",
     inactive: "#ddd"
   },
-  title: "Zalando Tech Radar — 2020.07",
+  title: "Runtastic Tech Radar — 2020.11",
```
Comment (andreaseger) [inline docs/index.html]: I'd aim for a December version given we need to check-in with some more teams before we really have a first version

File: docs/index.html [hunk 2]
```diff
@@ -285,489 +141,465 @@
...
         active: false,
-        link: "../infrastructure/tomcat.html",
+        link: "../infrastructure/open_nebula.html",
         moved: 0
       },
       {
         quadrant: 1,
-        ring: 0,
-        label: "ZMON",
+        ring: 2,
```
Comment (andreaseger) [inline docs/index.html]: I think all the "middle" rings(trial, assess) need to be rechecked, most are probably actually either in `adopt` or `hold`

File: docs/index.html [hunk 3]
```diff
@@ -797,21 +629,17 @@ <h3>What is the Tech Radar?</h3>
...
 
 <h3>How do we maintain it?</h3>
 
 <p>
-The Tech Radar is maintained by the <em>Zalando Technologists Guild</em> &mdash; an open group of  senior Zalando technologists committed to devote time to the Tech Radar purpose. The guild self organises to maintain the Tech Radar documents, including this public version. Assignment of technologies to rings is the outcome of ring change proposals, which are discussed and voted on in guild meetings. The Tech Radar depends on active participation and input from all engineering teams at Zalando.
-</p>
-
-<p>
-There is also an <a href="https://techradar.zalando.net/">internal version of the Tech Radar</a> that contains background information about each technology &mdash; when to use it, pros and cons, main risks, which teams at Zalando are using it, and for what purpose.
+The Tech Radar is maintained by the <em>Runtastic Architecture Guild</em> &mdash;. Assignment of technologies to rings is the outcome of ring change proposals, which are discussed and voted on in guild meetings. The Tech Radar depends on active participation and input from all engineering teams.
```
Comment (andreaseger) [inline docs/index.html]: I know you just rephrased it form the zalando one but I'd remove the stuff about `change proposals` and how we vote for things in the guild meetings. We actually first have to figure our process around this out and probably include other teams in this process

**Review by andreaseger — CHANGES_REQUESTED** [review]
I would add a WIP to the readme until we actually have a first version. Readme in general needs some updating, licence too.

File: docs/index.html [hunk 4]
```diff
@@ -285,489 +141,465 @@
...
       },
       {
         quadrant: 3,
-        ring: 3,
-        label: "HornetQ",
+        ring: 1,
+        label: "Databricks",
         active: false,
-        link: "../queues/hornetq.html",
+        link: "../databricks.html",
```
Comment (andreaseger) [inline docs/index.html:715]: I haven't tried this actually but I assume these links would work if the target is available. Maybe at least align the paths somewhat and create the empty folders in the repo already (with just a .gitkeep file for now)

File: docs/index.html (hunk 2 above)
Comment (rtgeha) [reply docs/index.html]: I agree - I would continue to work with the prototype on our board and then update the values here

File: docs/index.html (hunk 4 above)
Comment (rtgeha) [reply docs/index.html:715]: I think we first need to agree if we want the detail pages also public. I guess that Zalando has their detail pages in a separate private repository and has a separate build for the internal version

**Pull request merged**